include/utility/vpd_specific_utility.hpp

```diff
@@ -377,5 +377,24 @@ inline void getVpdDataInVector(const std::string& vpdFilePath,
         throw;
     }
 }
+
+/**
+ * @brief An API to get D-bus representation of given keyword name
+ *
+ * @param[in] i_keywordName - Keyword name.
+ *
+ * @return D-bus representation of keyword name.
+ */
+inline std::string getDbusKeywordName(const std::string& i_keywordName)
+{
+    // D-bus doesn't support "#". Replace "#" with "PD_" for those "#"
```

Comment:
would recommend to have a keyword size check.

Reply:
done